### ISSUE TITLE
Adding Transform All policy for queries incorrectly removes query parameters in the backendUrl

### ISSUE THREAD

**Issue by alexsong93** [openwhisk/apigateway#46]
According to the policy [docs](https://github.com/openwhisk/apigateway/blob/develop/doc/policies.md#reqmapping), to transform all query parameters to body, you would put this in policies:
```
{
    "action": "transform",
    "from": {
      "name": "*",
      "location": "query"
    },
    "to": {
      "name": "*",
      "location": "body"
    }
}
```
However, this also removes the query parameters in the backendUrl.

**Comment by DavidMGreen** [comment]
Correct. The query parameters are added to the query array at the beginning https://github.com/openwhisk/apigateway/blob/develop/api-gateway-config/scripts/lua/policies/mapping.lua#L63. The other query parameters were being added into the query array in addition to the backend query parameters. Is the `ngx.var.backendUrl` still being set?

**Comment by mhamann** [comment]
Is this now working as expected?
